recipes/flask-allows/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "flask-allows" %}
+{% set version = "0.4" %}
+{% set sha256 = "386d6535fd26fc610c4b000135ba66057fa743479ba6ef9d995de65c613c51a9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
```

Comment:
I think `flask` should be removed from `build` and added to the `run` requirements. The below you don't need to have a `requires` section under `test.

Reply:
Just to expand on my comment, any `run` requirement will be available during the testing. Further, `flask` isn't needed in the build as it is never imported by `setup.py` either directly or indirectly. The `--single-version-externally-managed` flag to `setup.py` makes it such that packages that are only listed in `install_requires` (https://github.com/justanr/flask-allows/blob/master/setup.py#L37) don't need to be `build` requirements.

Reply:
Done, thanks for the clarifications